Makefile

```diff
@@ -49,6 +49,18 @@ whitespace: ## Run fix-whitespace in check mode
 fix-whitespace: ## Run fix-whitespace in fix mode
 	fix-whitespace --verbose
 
+.PHONY: typos
```

Comment:
Do we want to consider adding this to `make check`?

Reply:
I was actually expecting a discussion, since it does add a non-Haskell dependency and not everyone will have a Rust ecosystem sitting around. (I do, as it turns out.)

Reply:
Good suggestion. I've added `typos` after `whitespace` and before `style` as it is a fast check, taking ~12s.

Reply:
The suggested fixes are show in the CI failure and these can be applied by hand, https://github.com/cabalism/cabal/actions/runs/12061197446/job/33632847383.

If needing rust stuff is a problem for some, we could remove `typos` as a dependency of the `checks` PHONY target in the makefile. The `checks` target is only used locally, isn't it (and not intended for CI)?

Reply:
Correct. It combines multiple checks which in CI are independent, only some of which use the `Makefile` targets (notably, I need to review what's up with the `hlint` CI setup before I will be willing to entrust it to the `Makefile`).